Standards/scs-xxxx-v1-sovereign-cloud-standards-yaml.md

```diff
@@ -127,11 +128,15 @@ iaas:
     standards:
       - name: Flavor naming
         url: https://raw.githubusercontent.com/SovereignCloudStack/Docs/main/Standards/SCS-0003-v1-flavor-naming.md
-        check_tool: flavor-name-check.py
+        check_tools: 
+	  - url: flavor-name-check.py
```

Comment:
This seems to be a tab-stop

Reply:
Looks like vi misconfiguration. Thanks for catching this one!